build.rs

```diff
@@ -57,20 +69,17 @@ fn main() {
         println!("cargo:rustc-cfg=host_can_test_sgx");
 
         if (!cfg!(feature = "disable-sgx-attestation"))
-            && std::path::Path::new(AESM_SOCKET).exists()
-            && std::fs::metadata(AESM_SOCKET)
-                .unwrap()
-                .file_type()
-                .is_socket()
+            && Path::new(AESM_SOCKET).exists()
+            && fs::metadata(AESM_SOCKET).unwrap().file_type().is_socket()
         {
             println!("cargo:rustc-cfg=host_can_test_attestation");
         }
     }
 
     #[cfg(all(target_os = "linux", target_arch = "x86_64"))]
```

Comment:
Or this one?

Reply:
No, because this is only valid when x86_64 linux is the host